dockerfiles/Samtools/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM bashell/alpine-bash:latest
+
+ARG version=1.9
+
+RUN apk update \
+ && apk upgrade \
+ && apk add --no-cache \
+ build-base \
+ zlib-dev \
+ bzip2-dev \
+ xz-dev \
+ ncurses-dev \
+ ca-certificates \
+ wget
+
+RUN wget -q https://github.com/samtools/samtools/releases/download/${version}/samtools-${version}.tar.bz2
+
+RUN mkdir /samtools-${version}
+
+RUN tar -C / -xjvf samtools-${version}.tar.bz2
+
+WORKDIR /samtools-${version}/
+
+RUN make
+
+RUN mv /samtools-${version}/samtools /bin/ \
+ && rm -rf /samtools*
+
+RUN apk del build-base zlib-dev ca-certificates wget
+
+RUN apk update \
+ && apk upgrade \
+ && apk add --no-cache python3
+
+ENTRYPOINT samtools
```

Comment:
my docker is a bit rusty- but why do we use `ENTRYPOINT` here and not above?

Reply:
I originally made this dockerfile for direct use, the others where made with Cromwell in mind. As it overrides any entry point defined I decided not to include one. Additionally programs like biobambam do not have a single entry point.